bin/pa11y-ci.js

```diff
@@ -51,6 +54,13 @@ program
 	)
 	.parse(process.argv);
 
+// Parse the args into valid paths using glob and protocolify
+const urls = globby.sync(program.args, {
+	// Ensure not-found paths (like "google.com"), are returned
+	nonull: true
+}).map(protocolify);
+console.log(program.args, urls);
```

Comment:
This `console.log` is failing the tests at the moment, I think it's left in from debugging?